README.md

```diff
@@ -30,6 +30,20 @@ to avoid duplicates.
 Contributions are always welcome. The preferred way to receive code contribution is by submitting a 
 Pull Request on github.
 
+## Developing
+
+1. Clone the [ArduinoCore-API](https://github.com/arduino/ArduinoCore-API) repo to any convenient location:
+   ```
+   git clone https://github.com/arduino/ArduinoCore-API.git
+   ```
+1. Create an `<SKETCHBOOK>/hardware/arduino-git` folder, where `<SKETCHBOOK>` is the location of your
+  Arduino sketchbook.
+1. Change directories: `cd <SKETCHBOOK>/hardware/arduino-git`.
+1. Clone this repo: `git clone https://github.com/arduino/ArduinoCore-samd.git samd`.
+1. Change directories: `cd samd/cores/arduino`.
+1. Copy or symlink the `api` folder from the [ArduinoCore-API](https://github.com/arduino/ArduinoCore-API) repo (i.e. `ln -s <ARDUINO_CORE_API>/api .`, where `<ARDUINO_CORE_API>` is the location where you've cloned the ArduinoCore-API repository to).
```

Comment:
````suggestion
1. Change directories:
   ```
   cd <SKETCHBOOK>/hardware/arduino-git
   ```
1. Clone this repo:
   ```
   git clone https://github.com/arduino/ArduinoCore-samd.git samd
   ```
1. Change directories:
   ```
   cd samd/cores/arduino
   ```
1. Copy or symlink the `api` folder from the [ArduinoCore-API](https://github.com/arduino/ArduinoCore-API) repo:
   ```
   ln -s <ARDUINO_CORE_API>/api .
   ```
   where `<ARDUINO_CORE_API>` is the location where you've cloned the ArduinoCore-API repository to.
````

Following the acceptance of my proposal to use a fenced code block for the command in step (1), I extend it to the other steps here.

The copy/paste benefit is perhaps a bit less in the commands that use a placeholder, but I think it is still worthwhile even if only for the sake of consistency with the formatting of the other steps.